src/Directives.jl

```diff
@@ -95,11 +99,20 @@ function parsebracket end
 parsebracket(text :: AbstractString) = parsebracket(getdirective(text)...)
 parsebracket{D}(:: Directive{D}, text) = error("Unknown directive: '$D'")
 
+function parsebracket{D}(TD :: Directive{D}, arg, text)
+    if strip(arg) == ""
+        parsebracket(TD, text)
+    else
+        error("parsebracket not defined for directive '$D' with positional argument.")
+    end
+end
+
 function parsebracket(:: directive"docs", text)
+    eval(Expr(:toplevel, Expr(:using, :Lexicon)))
```

Comment:
> but maybe I'll warm to it

Yeah, I'm still very willing to change it to something better if we can find it.

> Is the idea that any markdown files in docs/src will be built to docs/build with the expansions?

Yes, any ``.md`` file in ``source`` will be read, have directives executed, and finally written to ``build``.

> we should lob some default css + js (for latex) in there.

That sounds fine to me.

> I think generating a README is something that'll be really useful.

Currently ``makedocs`` won't build anything to outside of ``build``. I guess setting ``README.md`` as a link to ``doc/build/README.dm`` would work? Not sure how github handles links.